scripts/parse-tsdocs.js

```diff
@@ -5,22 +5,29 @@ const options = {
   savePropValueAsString: true,
 };
 
-const parsedPackages = ['radio-group'];
+const parsedPackages = [
```

Comment:
Can you explain a little whats going on here? I'm not sure I follow why it was only `radio-group` before and now its only `palette`?

Reply:
This script was still just considered a "draft" in this PR. PR #1264 has for what will probably be used moving forward!

I think it would be best for us to merge #1264, then we can pull in the changes to `parse-tsdocs` here. What's really important in this PR is the change to palette.